### ISSUE TITLE
Error when creating relations to contenttypes without explicitly defined order.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#5954]
On a set up with "strict errors", I get the following, when creating a new 'block'

![screen shot 2016-10-25 at 17 19 15](https://cloud.githubusercontent.com/assets/1833361/19692159/2ec4fe7c-9ad7-11e6-8961-ebe9a669ec92.png)

Full stack trace below.

## Steps to reproduce

Create a blocks contenttype: 

```
blocks:
    name: Blocks
    singular_name: Block
    fields:
        title:
            type: text
            class: large
            group: "Block"
        slug:
            type: slug
            uses: [ title ]
    relations:
        pages:
            multiple: false
```

Update the db, and click "new block" to trigger the error. 

Adding an order to the CT makes the error go away: 

```
    relations:
        pages:
            multiple: false
            order: id
```

-------

```
# 1	Symfony\Component\Debug\ErrorHandler::handleError(8, "Undefined index: order", "/Users/bob/Sites/bolt-origin/ …", 191, [array]) 
[root]/src/Storage/ContentRequest/Edit.php # line 191
# 2	Bolt\Storage\ContentRequest\Edit::getRelationsList([array]) 
[root]/src/Storage/ContentRequest/Edit.php # line 164
# 3	Bolt\Storage\ContentRequest\Edit::action(Content, [array], [false]) 
[root]/src/Controller/Backend/Records.php # line 97
# 4	Bolt\Controller\Backend\Records::edit(Request, "blocks", "") 
# 5	call_user_func_array([array], [array]) 
[root]/vendor/symfony/http-kernel/HttpKernel.php # line 144
# 6	Symfony\Component\HttpKernel\HttpKernel::handleRaw(Request, 1) 
[root]/vendor/symfony/http-kernel/HttpKernel.php # line 64
# 7	Symfony\Component\HttpKernel\HttpKernel::handle(Request, 1, [true]) 
[root]/vendor/silex/silex/src/Silex/Application.php # line 586
# 8	Silex\Application::handle(Request) 
[root]/vendor/silex/silex/src/Silex/Application.php # line 563
# 9	Silex\Application::run(Request) 
[root]/src/Application.php # line 93
# 10	Bolt\Application::run() 
[root]/index.php # line 12
```

**Comment by GawainLynch** [comment]
@rossriley Do you have a second to look at this? … kinda flooded here atm

**Comment by bobdenotter** [comment]
I can whip up a fix, I think. :-)

**Comment by bobdenotter** [comment]
Fixed by #5955
